test/E2ETests/SmokeTestsOnNanoServer.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
```

Comment:
Most(none) of the files in MusicStore do not have a copyright header. Probably we can do an repo wide clean up later.